test/mocks/config/mocks.h

```diff
@@ -70,13 +71,23 @@ class MockGrpcMux : public GrpcMux {
 class MockGrpcMuxCallbacks : public GrpcMuxCallbacks {
 public:
   MockGrpcMuxCallbacks();
-  virtual ~MockGrpcMuxCallbacks();
+  virtual ~MockGrpcMuxCallbacks() = default;
```

Comment:
These should actually be concrete and in the `.cc` for compile performance reasons; see https://github.com/google/googletest/blob/master/googlemock/docs/CookBook.md#making-the-compilation-faster

Reply:
Oh! Thanks, good to know!